packages/quantic/force-app/main/default/lwc/quanticCategoryFacetValue/quanticCategoryFacetValue.js

```diff
@@ -52,6 +52,15 @@ export default class QuanticCategoryFacetValue extends LightningElement {
     return this.item.value;
   }
 
+  get ariaLabelValue() {
+    return `Inclusion filter on ${this.facetValue};`;
```

Comment:
Is the trailing semicolon mandatory?

```suggestion
    return `Inclusion filter on ${this.facetValue}`;
```

Reply:
No, removing it.